salt/version.py

```diff
@@ -96,107 +96,107 @@ class SaltStackVersion(object):
         'Nitrogen'      : (MAX_SIZE - 102, 0),
         'Oxygen'        : (MAX_SIZE - 101, 0),
         'Fluorine'      : (MAX_SIZE - 100, 0),
+        'Neon'          : (MAX_SIZE - 99, 0),
```

Comment:
Without it, the calls to ``salt.utils.warn_until()`` I added will throw a ValueError since ``Neon`` is not defined in version.py. For example, try with a much later release codename:

```python
>>> import salt.utils
>>> salt.utils.warn_until('Lead', 'Message')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/erik/git/salt/main/salt/utils/__init__.py", line 2227, in warn_until
    version = salt.version.SaltStackVersion.from_name(version)
  File "/home/erik/git/salt/main/salt/version.py", line 264, in from_name
    'Named version \'{0}\' is not known'.format(name)
ValueError: Named version 'Lead' is not known
```